imagetest/fixtures.go

```diff
@@ -133,3 +133,22 @@ func (t *TestVM) EnableSecureBoot() {
 		}
 	}
 }
+
+// ResizeDisk resize the disk of the current test VMs in workflow.
+func (t *TestVM) ResizeDisk(diskSize int) error {
+	// Grab the wait step that was added with CreateTestVM.
+	waitStep, ok := t.testWorkflow.wf.Steps["wait-"+t.name]
+	if !ok {
+		return fmt.Errorf("wait-%s step missing", t.name)
+	}
+
+	diskResizeStep, err := t.testWorkflow.addResizeDisk(t.name, t.name, int64(diskSize))
+	if err != nil {
+		return err
+	}
+
+	if err := t.testWorkflow.wf.AddDependency(diskResizeStep, waitStep); err != nil {
```

Comment:
you add the new step and with this dependency, you ensure it will run after the VM is done running the test. but in this test you want to boot VM -> run test -> resize disk -> reboot VM -> run test -> wait for test completion. how will you guarantee the reboot happens *after* the resize?

when you're developing new fixtures, use the -print option to see what the resulting workflow looks like.